conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -229,6 +229,9 @@ def items(self):
         # First, let's load all the components PC files
         # Loop through all the package's components
         for comp_ref_name, comp_cpp_info in self._dep.cpp_info.get_sorted_components().items():
+            should_skip = self._get_property("pkg_config_skip", self._dep, comp_ref_name, check_type=bool)
+            if should_skip:
```

Comment:
Could we use the `pkg_config_name` property instead? Something like:

```suggestion
            should_skip = self._get_property("pkg_config_name", self._dep, comp_ref_name)
            if should_skip == "none":
```
So, it'd be a similar mechanism as it's done for CMake cases

Reply:
> So, it'd be a similar mechanism as it's done for CMake cases

TIL